client/src/pages/Review/Review.tsx

```diff
@@ -36,19 +56,9 @@ const Review = () => {
     onMainMarginLeft();
     window.addEventListener("resize", onMainMarginLeft);
     return () => window.removeEventListener("resize", onMainMarginLeft);
-  }, [navRef.current, mainRef.current, currentCourseKey]);
```

Comment:
`currentCourseKey` or its equivalent variable shouldn't be removed. 

`NavigationPanel` and `MainPanel` might occur display bug when screen is resized and without also monitoring course tab change.

This is a fix to the mentioned bug that was pointed out in a previous PR.

Reply:
The bug is no longer reproducible as far as I can tell

Reply:
Oh, it came back I'll squash it now

Reply:
Even better! Then remove it